js/app.js

```diff
@@ -1,271 +1,82 @@
 
-// Alki Store
-var alkiStore = {
-  maxCustomers: 16,
-  minCustomers: 2,
-  avgCookiesPerSale: 4.6,
+  var trElement = document.createElement('tr'); // create 2nd row
+  var tdElement = document.createElement('td'); // create data cell
 
-  // generate random # of customer with function and return it rounded
-  randomCustomer: function() {
-    return Math.round(Math.random() * (this.maxCustomers - this.minCustomers) + this.minCustomers);
-  },
-  // round random customer and multiply by average cookies per customer/sale
-  averagePerHour: function() {
-    return Math.round(this.randomCustomer() * this.avgCookiesPerSale);
-  },
+  tdElement.textContent= this.storeName; // store name goes in the first cell
+  tdElement.appendChild(trElement); // append to parent
 
-  // make list for HTML
-  makeList: function() {
-    var alkiUlEl = document.getElementById('capitolHillStore');
-    var alkiTotal = 0;
+  for(var i = 0; i < storeLocations; i++) {
+    tdElement = document.createElement('td');
+    tdElement.textContent = this.hourlyCookies;
+    trElement.appendChild(tdElement);
+  }
 
-    // use for loop
-    for(var i = 0; i < hoursOperation.length; i++) {
-    // create list elements
-      var alkiLiEl = document.createElement('li');
-      // give it content
-      var alkiCookies = this.averagePerHour();
-      alkiLiEl.textContent = hoursOperation[i] + ': ' + alkiCookies + ' cookies';
-      alkiTotal += alkiCookies; 
-      // append to the DOM -> parentElement.appendChild(childElement);
-      alkiUlEl.appendChild(alkiLiEl);
-    }
+  tdElement = document.createElement('td');
+  tdElement.textContent = this.totalCookies();
+  trElement.appendChild(tdElement);
 
-    var totalCookiesEl = document.createElement('li');
-    alkiUlEl.appendChild(totalCookiesEl);
-    totalCookiesEl.textContent = 'Total: '+ alkiTotal;
-  },
+  storeTable.appendChild(trElement);
+  console.log('this is the table row element', trElement);
 };
 
-// Call the functions
-pikeStore.makeList();
-seatacAirportStore.makeList();
-seattleCenterStore.makeList();
-capitolHillStore.makeList();
-alkiStore.makeList();
-
-
-// Method: a function is a property of an object and must be invoked using do notation, such as firstAndPike.randomCustPerHour()
-
-// This is how to define a function that is NOT a method - function randomCustPerHour() {}
+//========================================================================================================================
+//========================================================================================================================
+// CALLING FUNCTIONS/METHODS==============================================================================================
+new Stores('1st and Pike', 23, 65, 6.3);
+new Stores('SeaTac Airport', 24, 3, 1.2);
+new Stores('Seattle Center', 38, 11, 3.7);
+new Stores('Capitol Hill', 38, 20, 2.3);
+new Stores('Alki', 16, 2, 4.6);
+Stores.makeHeader();
+Stores.prototype.renderRow();
```

Comment:
I suspect there might be an issue with how you are invoking your renderRow method here. You shouldn't need to include prototype.